shell/platform/darwin/graphics/FlutterDarwinContextMetalSkia.mm

```diff
@@ -111,6 +112,14 @@ - (void)dealloc {
                                                           enableImpeller:NO];
 }
 
+- (CVMetalTextureCacheRef)textureCache {
+  return _textureCache;
+}
+
+- (void)setTextureCache:(CVMetalTextureCacheRef)cache {
+  _textureCache.Reset(cache);
```

Comment:
Is `Reset` correct, or should it be `Retain`? I don't actually know offhand what the semantics of a CFTypeRef `@property` is; naively I would expect it to be like an Obj-C object property, where the setter adds its own ownership rather than taking over ownership.

Reply:
Actually turns out that upon checking the header, the property was `readonly` so... deleted. 

Your interpretation seems entirely like what I'd have expected though, so the original code erroneous and were we to have kept this, it should have been switched to Retain.